src/App.jsx

```diff
@@ -157,6 +163,21 @@ function App() {
     auth
       .createUserWithEmailAndPassword(email, password)
       .then((authUser) => {
+        const imageRef = ref(storage, "image");
```

Comment:
can you tell me what you are trying to do here? Why are you uploading image in advance to the uploadTask function? won't it be duplicate?

Reply:
@EdkoPitonak please look, so I can proceed

Reply:
Well in my understanding actually in uploadTask you are not uploading the image because it is not working, i upload in uploadBytes , and in uploadTask i just update the profil with url i get in getDownloadUrl(imageRef)

Reply:
Okay Eduard, but actually we're uploading in uploadTask through the put method and when it gets executed, we add the URL with the username. I don't think we need to repeat that, or perhaps I might be wrong, but right now can we focus on the main issue #67 and solve that only for now? we can create another bug issue for this one I guess. Thanks

Reply:
Okay , iam sorry then , so i will delete these changes and let only the fix for the main issue then and then send the code for review i guess.

Reply:
No need to be sorry Ed, you were doing great, it's just that our current requirements didn't match. Thanks a lot for your patience though. I'll be waiting for the update.